pkg/cli/stats.go

```diff
@@ -53,11 +53,8 @@ func stats(c *cli.Context) error {
 	}
 
 	if c.Bool("json") {
-		if err := json.NewEncoder(os.Stdout).Encode(result); err != nil {
-			return err
-		}
-
-		return nil
+		err := json.NewEncoder(os.Stdout).Encode(result)
```

Comment:
Let's skip an unnecesary variable :)
```suggestion
		return json.NewEncoder(os.Stdout).Encode(result)
```